docs/Generators/RocketChip.rst

```diff
@@ -0,0 +1,66 @@
+RocketChip
+==========
+
+RocketChip is an SoC generator supported by SiFive. Chipyard uses RocketChip
+as the basis for producing a RISC-V SoC including Rocket, BOOM, and/or Hwacha.
```

Comment:
Maybe it's worth mentioning here explicitly the difference between the RocketChip generator and the Rocket core?